recipes/flexx/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "4.0.0rc1" %}
+
+package:
+  name: example
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/zoofIO/flexx.git
+  git_tag: b824bf9f9ab7d441b492d42484bb88877cd4af93
```

Comment:
Change this to:

```yaml
source:
  fn: flexx-{{ version }}.tar.gz
  url: https://github.com/zoofIO/flexx/archive/v{{ version }}.tar.gz
  sha256: eeff10a64f076a65b0762d1d7082e825e3ffb6bf0aca15150533b1bb9ccb70a5
```

Note that I used release `0.3.1` as we don't have a proper system for release candidates yet. (Although, as soon as the feedstock is created I encourage you submit a PR for the `rc`.)